src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -618,19 +618,19 @@ public FabricEvents.ExtensionsEvents TraceSource
 
             this.ThrowIfClosed();
 
-            // set the incoming callContext as the new logical call context, before
+            // Sets the incoming callContext as the new logical call context, before
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.